.github/workflows/ci_linux_x64_clang.yml

```diff
@@ -28,37 +28,52 @@ jobs:
   linux_x64_clang:
     needs: setup
     if: contains(fromJson(needs.setup.outputs.enabled-jobs), 'linux_x64_clang')
-    runs-on:
-      - self-hosted # must come first
-      - runner-group=${{ needs.setup.outputs.runner-group }}
-      - environment=${{ needs.setup.outputs.runner-env }}
-      - cpu
-      - os-family=Linux
+    runs-on: azure-linux
+    container: 
+      image: ghcr.io/iree-org/cpubuilder_ubuntu_jammy_x86_64@sha256:ba10d33ef502d8b33aaa43800e640a6866983ef353b73fda2ed6595c4275e066
+      options: --privileged
```

Comment:
Should add a comment here on why this is needed (blobfuse2 needs it?)

Comment could match this style: https://github.com/iree-org/iree/blob/9dc6b904f9125c8e407ef0a23b81e93c0a4757d1/.github/workflows/ci_linux_x64_clang_tsan.yml#L35-L40